server/graphql/types/src/types/vaccine_course_item.rs

```diff
@@ -12,6 +12,11 @@ impl VaccineCourseItemNode {
     pub async fn id(&self) -> &str {
         &self.row().id
     }
+
+    pub async fn item_id(&self) -> &str {
+        // TODO Look up item_id
```

Comment:
```suggestion
        // TODO Look up item_id https://github.com/msupply-foundation/open-msupply/issues/4129
```
Adding the issue number to make it easier to find the places we need to consider :)

Reply:
I've added the loader in here.

Wondering if that is all that is needed? I see that the loader generates the item based on item_link_id rather than the item_id.